src/tests/ftest/scrubber/aggregation.py

```diff
@@ -0,0 +1,56 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithAggregation(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
+    """Inject Checksum Fault with scrubber enabled
+    and scrubber threshold set to a certain value.
+    Aggregation is run on the background.
+
+    :avocado: recursive
+    """
+    def test_target_eviction_during_aggregation(self):
+        """JIRA ID: DAOS-7333
+
+        1. Start the Aggregation task.
+        2. Create checksum faults above scrubber threshold
+        and see whether SSD auto eviction works as expected.
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=scrubber,faults
+        :avocado: tags=TestScrubberEvictWithAggregation,test_target_eviction_during_aggregation
+        """
+        initial_metrics = {}
+        final_metrics = {}
+        self.add_pool()
+        # Disable the aggregation on the pool.
+        self.pool.set_property("reclaim", "disabled")
+        self.add_container(self.pool)
+        # Pool and Containers are already created. Just run the IOR.
+        self.run_ior_with_pool(create_cont=False)
+        # Enable the aggregation on the pool.
+        self.pool.set_property("reclaim", "time")
+        self.pool.set_property("scrub", "timed")
```

Comment:
If you set aggregation and scrubber to time, does it take less than 120 sec to start/run? i.e., how do you know sleeping 120 sec is enough? IMO, it would better to periodically check the aggregation/scrubber status rather than sleeping for the fixed amount of time.

Reply:
@shimizukko: Unfortunately, we don't have a mechanism to validate whether Aggregation is happening with status flags... Regarding scrubber the metrics information change which gives us an idea scrubber is working in the background.
Do you have any suggestions regarding the aggregation? I thought 2 minutes is a lot of time for either Aggregation to start... Am I wrong?

Reply:
I looked up how to check whether aggregation has started and thought `engine_pool_vos_aggregation_obj_deleted` would work, but I believe your metric also works. For scrubber, you're using `scrub-freq:1`, so I think you just need to wait for a few sec and don't really have to check the metric.